tests/test_utils.py

```diff
@@ -993,3 +993,18 @@ def test_model_to():
     model = torch_models.resnet18()
     model = misc.model_to(on_gpu=False, model=model)
     assert isinstance(model, nn.Module)
+
+
+def test_save_json():
+    """Test for placing model on device."""
```

Comment:
Is this test for model or model output?

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/109/commits/0e5e72498ed711e569a359ff33dfa746ba73e6e1